spec/controllers/ems_cloud_controller_spec.rb

```diff
@@ -271,4 +273,64 @@
       controller.send(:update_ems_button_validate, mocked_ems)
     end
   end
+  describe "#test_toolbars" do
+    before do
+      server
```

Comment:
https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let

use `let!` instead of `let` here and avoid the `server` line? maybe?